diff --git a/config.mk b/config.mk
index 3b32b7c..87959c6 100644
--- a/config.mk
+++ b/config.mk
@@ -11,10 +11,10 @@ X11INC = /usr/X11R6/include
 X11LIB = /usr/X11R6/lib
 
 # flags
-CPPFLAGS = -I$(X11INC) -D_DEFAULT_SOURCE
-CFLAGS   = -std=c99 -ffunction-sections -fdata-sections -pedantic -Wall -Wextra -Os
-LDFLAGS  = -L$(X11LIB) -s
-LDLIBS   = -lX11
+CPPFLAGS += -I$(X11INC) -D_DEFAULT_SOURCE
+CFLAGS   += -std=c99 -ffunction-sections -fdata-sections -pedantic -Wall -Wextra
+LDFLAGS  += -L$(X11LIB) -s
+LDLIBS   += -lX11
 
 # compiler and linker
-CC = cc
+# CC = cc
-- 
2.26.2

